src/main/java/core/basesyntax/service/impl/ShopServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class ShopServiceImpl implements ShopService {
+    private OperationStrategy strategy;
+    private Storage storage;
+
+    public ShopServiceImpl(OperationStrategy strategy, Storage storage) {
+        this.strategy = strategy;
+        this.storage = storage;
+    }
```

Comment:
Avoid beginning class or method implementation with an empty line. Make sure to remove any redundant empty lines.